maintain.js

```diff
@@ -1,25 +1,345 @@
-/* Copyright © 2021 Ríona Stokes and other contributors, MIT License. */
-'use strict'
+module.exports = () => {
 
-const Fs = require('fs')
+  const Filehound = require('filehound')
+  const Path = require('path')
+  const Fs = require('fs')
+  const Hoek = require('@hapi/hoek')
+  const Marked = require('marked')
 
+  const checkList = require('./checks')
```

Comment:
prefer import style:

```
const { checkList } = require('./checks')
```

```
function checkList { ... }
module.exports = {
  checkList
}
```

Reply:
Modification to require resolved with commit 8ffd82fb6a786b28be7c2e95fd85e53de929fd9c

Need clarification on second block - module.exports to be put into checks.js as well ?